### ISSUE TITLE
Terminal drop feedback should not show when dragging into itself

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#125420]
Repro:

1. Open a terminal
2. Drag tab into the actual terminal, feedback should not show

![image](https://user-images.githubusercontent.com/2193314/120689146-a6d3da00-c458-11eb-8739-61c01e53e368.png)

**Comment by meganrogge** [comment]
the editor does this too. @bpasero @sbatten thoughts?

![recording - 2021-06-10T123229 427](https://user-images.githubusercontent.com/29464607/121585853-e74acf00-c9e7-11eb-8f87-bdf60fe33319.gif)

**Comment by sbatten** [comment]
that is true because you have multiple tabs open and you can split with that editor. In the terminal case, you have multiple terminals within tabs, but in editors you have multiple tabs within groups

**Comment by meganrogge** [comment]
Yeah that makes sense. 

We shouldn't show the hover feedback when there's only one terminal in a tab and that's the one being dragged. 

When a split is being dragged into the group to which it belongs, we should still show the hover feedback to allow for re-arranging.

**Comment by bpasero** [comment]
The drop feedback shows to indicate that you can split the editor top, bottom, left or right. The full drop feedback is mainly a way to basically drop with no-op. I never heard complaints about this.
